### PR TITLE
Fix typo in Traditional Chinese translations

### PR DESCRIPTION
**Pull request by ydFu**
* The commit fixed some words ,sentences and typo for Traditional Chinese translations

Signed-off-by: ydFu <ader.ydfu@gmail.com>

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>

> /kind optimization

**What this PR does / why we need it**:

* The commit fixed some words ,sentences and typo for Traditional Chinese translations

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for reviewers**:
```
```

**Additional documentation, usage docs, etc.**:

```docs

```

### CODE REVIEW

File: locales/tc/alerting.js [hunk 1]
```diff
@@ -116,7 +116,7 @@ module.exports = {
   CONSECUTIVE_OPTIONS: '連續 {value} 次',
 
   MAX_SEND_NOT_LIMIT: '不限制',
-  MAX_SEND_COUNT: '最多重發{count}次',
+  MAX_SEND_COUNT: '最多重發 {count} 次',
```
Comment (LinuxSuRen) [inline locales/tc/alerting.js:119]: Thanks for providing this improvement. I'm wondering if you mind proposing a specification about the expression. For example, when we should keep the space between the word. I believe that it's good for keeping the same style in the future.

Comment (ydFu) [reply locales/tc/alerting.js:119]: Thank for your reply.
I just a littel contributer , but I suggest to define a specification about the expression this easy to management,too.

For example:
1. Keep the space between the english word.
2. Punctuation Marks on the localization.
etc...

I know this is writing style not writing standard in the different people have different opinions.

Comment (LinuxSuRen) [reply locales/tc/alerting.js:119]: /cc @kubesphere/sig-docs

Comment (FeynmanZhou) [reply locales/tc/alerting.js:119]: See the [localization style guide](https://github.com/kubesphere/website/blob/master/localization_style_guides/KubeSphere%20Localization%20Style%20Guide%20(for%20Simplified%20Chinese).md) for specification.

Comment (Sherlock113) [reply locales/tc/alerting.js:119]: Thanks for the PR! We have a style guide for Simplified Chinese. I am not sure if Traditional Chinese follows the same way, but perhaps you can have a look at the style guide first to see if these rules fit for Traditional Chinese. As we have the Traditional Chinese loc on the console, I think it is necessary that we have a Traditional Chinese style guide going forward.

Comment (ydFu) [reply locales/tc/alerting.js:119]: Very constructive discussion that learn a lot for me.

I had a look at the style guide to recheck this PR. If there has any question, please let me know right away. Thanks.

Thank for @LinuxSuRen @FeynmanZhou @Sherlock113

**Comment by FeynmanZhou** [comment]
Actually, I didn't see any problems with this PR, why do you close it? 

: )

**Comment by ydFu** [comment]
@FeynmanZhou
Oop, Please review again. thank.

**Pull request merged**